src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -113,32 +113,28 @@ class NotebookCard extends Component {
       ])
     ])
 
-    const jupyterIcon = icon('jupyterIcon', {
-      style: {
-        height: 125,
-        width: 'auto',
-        color: colors.gray[5]
-      }
-    })
-
     const title = div({
       title: printName(name),
       style: _.merge({
-        ...Style.elements.card.title,
-        textOverflow: 'ellipsis', whiteSpace: 'nowrap'
+        ...Style.elements.card.title, whiteSpace: 'normal', overflowY: 'auto'
       }, listView ? {
         marginLeft: '1rem'
-      } : undefined)
+      } : { height: 60, padding: '1rem' })
     }, printName(name))
 
     return a({
       href: notebookLink,
-      style: {
+      style: _.merge({
         ...Style.elements.card.container,
         ...notebookCardCommonStyles(listView),
-        flexShrink: 0,
-        alignItems: listView ? 'center' : undefined
-      }
+        flexShrink: 0
+      }, listView ? {
```

Comment:
nit: There's already one conditional in `notebookCardCommonStyles`, we should be able to extend that rather than adding another one.